src/main/scala/cognite/spark/v1/EventsRelation.scala

```diff
@@ -2,15 +2,15 @@ package cognite.spark.v1
 
 import cats.effect.{ContextShift, IO}
 import cats.implicits._
-import com.cognite.sdk.scala.v1.{Event, EventUpdate, EventsFilter, GenericClient, TimeRange}
+import com.cognite.sdk.scala.v1.{Event, EventCreate, EventUpdate, EventsFilter, GenericClient, TimeRange}
```

Comment:
Looks like `TimeRange` isn't used anymore. Not related to this PR, but let's remove it anyways.